packages/fontless/docs/content/index.md

```
@@ -181,6 +186,33 @@ fontless({
 })
 ```
 
+## Glyph Subsetting
```

markdownlint-cli2 (0.23.2)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)